modules/core/server/routes/core.server.routes.js

```diff
@@ -22,6 +22,9 @@ module.exports = function(app) {
   // Return a 404 for all undefined api, module or lib routes
   app.route('/:url(api|modules|lib|developers)/*').get(core.renderNotFound);
 
+  // Gives the service worker access to any config it needs
+  app.route('/config/sw.js').get(core.renderServiceWorkerConfig);
```

Comment:
The service worker does not have access to the HTML/DOM. It's very restricted.

Perhaps a better option is sending a message to it once it's started up [1]. I'm not very happy with the current approach.

[1] http://craig-russell.co.uk/2016/01/29/service-worker-messaging.html